storage/client_raft_test.go

```diff
@@ -620,5 +620,12 @@ func TestReplicateAddAndRemove(t *testing.T) {
 		// TODO(bdarnell): when we have GC of removed ranges, verify that
 		// the downed node removes the data from this range after coming
 		// back up.
+
+		// Wait out the leader lease to make the range GC'able.
+		mtc.manualClock.Increment(int64(storage.DefaultLeaderLeaseDuration) + 1)
```

Comment:
This test is in package storage_test, so we'd need an exported accessor for the lease. Do we want to add that?